src/ts/Html5Metadata.ts

```diff
@@ -59,7 +64,7 @@ export class Html5Metadata implements Conviva.MetadataInterface {
 
   // Relying on HTTP user agent string parsing on the Conviva Platform.
   getDeviceCategory(): Conviva.Client.DeviceCategory {
-    return null;
+    return this.deviceCategory || Conviva.Client.DeviceCategory.WEB;
```

Comment:
Let's move the default value into `ConvivaAnalytics`. We have a `// Set default config values` in the constructor of `ConvivaAnalytics`. We can set the config option to `WEB` there.